### DISCUSSION TITLE
Maximum request body size 1048576 exceeded, actual body size 1049648

### DISCUSSION THREAD

**Discussion by Th0rgal** [aio-libs/aiohttp#6019] [Q&A] [answered]
## Long story short
I'm trying to uploads small files (less than 120MiB) to my server written with aiohttp but when they are a little too bigs it doesn't work.

## Expected behaviour
The request should be handled.

## Actual behaviour
The server returns:
Maximum request body size 1048576 exceeded, actual body size 1049648

## Steps to reproduce
Clone my small server polymath: https://github.com/Th0rgal/Polymath/blob/master/server.py
Perform: 
curl -F "pack=@./file.zip" -F "id=666599" -X POST http://localhost:8080/upload                    

## Your environment
My server runs under nixos 19.09 (linux 5.4.6) and I get this error with any clients (windows, ubuntu, nixos, etc).

Is there a way to change this limit server-sidely?

**Comment by hh-h** [comment, marked as answer]
> **client_max_size** – client’s maximum size in a request, in bytes. If a POST request exceeds this value, it raises an HTTPRequestEntityTooLarge exception.

https://aiohttp.readthedocs.io/en/stable/web_reference.html#aiohttp.web.Application

**Comment by Th0rgal** [comment]
> > **client_max_size** – client’s maximum size in a request, in bytes. If a POST request exceeds this value, it raises an HTTPRequestEntityTooLarge exception.
> 
> https://aiohttp.readthedocs.io/en/stable/web_reference.html#aiohttp.web.Application

Thank you!

**Comment by hh-h** [comment]
Yw, but issues not for questions, see: https://github.com/aio-libs/aiohttp/issues/4257

**Comment by zyb8543d** [comment]
> > > **client_max_size** – client’s maximum size in a request, in bytes. If a POST request exceeds this value, it raises an HTTPRequestEntityTooLarge exception.
> > 
> > 
> > https://aiohttp.readthedocs.io/en/stable/web_reference.html#aiohttp.web.Application
> 
> Thank you!

I met the same problem，and I set “client_max_size“，but it not work. do you have solve the problem？
